provisioning/standalone/README.md

```diff
@@ -0,0 +1,25 @@
+# Standalone Applications
+
+## What is a Standalone Application?
+
+**Standalone applications** are autonomous web services that are exposed by **HTTP**. Those services have to be packaged in a **container**.
+An example could be a **container** exposing a **static web page**. If the **container** includes **all the dependencies**, it can be considered a **standalone application**. Some examples are [listed below](#applications).
```

Comment:
```suggestion
An example could be a **container** exposing a **static web page**. If the **container** includes **all the dependencies**, it can be considered a **standalone application**. Some additional examples are [listed below](#applications).
```